page.py

```diff
@@ -12,7 +12,8 @@
 # Free Software Foundation, Inc., 59 Temple Place - Suite 330,
 # Boston, MA 02111-1307, USA.
 
-
+import gi
+gi.require_version('Gtk', '3.0')
```

Comment:
You've already added a `gi.require_version` in `LetterMatch.py` and you need just one.

Reply:
Oh ok, I will remove that